data/pigui/modules/system-view-ui.lua

```diff
@@ -814,11 +814,11 @@ function Windows.objectInfo:Show()
 				{ name = lc.RADIUS, icon = icons.body_radius,
 					value = (not starport) and ui.Format.Distance(body.radius) or nil },
 				{ name = lc.SURFACE_GRAVITY, icon = icons.body_radius,
-					value = (not starport) and ui.Format.Speed(body.gravity, true).." ("..ui.Format.Gravity(body.gravity / 9.80665)..")" or nil },
+					value = (not starport) and ui.Format.Speed(body.gravity, true).."²".." ("..ui.Format.Gravity(body.gravity / 9.80665)..")" or nil },
 				{ name = lc.ESCAPE_VELOCITY, icon = icons.body_radius,
 					value = (not starport) and ui.Format.Speed(body.escapeVelocity , true) or nil },
 				{ name = lc.MEAN_DENSITY, icon = icons.body_radius,
-					value = (not starport) and ui.Format.Mass(body.meanDensity).."/m3" or nil },
+					value = (not starport) and ui.Format.Mass(body.meanDensity).."/m³" or nil },
```

Comment:
@sturnclaw I looked into changing the mean density to **g/cm** in this PR but I think this would need some work. Right now we're (**I** really as I'm the one responsible for the original addition) just using **Format.Mass** which is in tons and stick a **/m³** to the end. To change this to **g/cm** we would need something like a new **Format.Density** to call. Also, instead of using Format.Mass, one could just use **body.meanDensity** and fix the format manually. Maybe I missed a way but I think adding a new **Format.Density** is the best way to do it.

Reply:
Adding Format.Density would be fine, as would changing the line to be `ui.Format.Number(body.meanDensity, 3) .. lc.UNIT_DENSITY` with `UNIT_DENSITY` being defined as the appropriate `g/m^3`. Regardless of the tack taken, the unit suffix should ideally be a localized string for consistency (as unlikely as it is for SI units to be written differently in other languages...)

Reply:
OK. The SI unit for density (Wikipedia) is kg/m3 and that's what you get from calcMeanDensity. You need to divide with 1000 (like I did in the editor) so you get **ui.Format.Number(body.meanDensity / 1000.0, 3)**
I'll poke around some more with it.

Reply:
I switched to SI and kg/m3 instead. It made the code cleaner and it feels more like a unit that miners and fixers in space a thousand years from now would use.